lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -32,6 +32,8 @@ patch.facecolor        : b
 patch.edgecolor        : k
 patch.antialiased      : True    # render patches in antialiased (no jaggies)
 
+hatch.linewidth        : 1.0
```

Comment:
No, I think unify on one is an ok complete break. We should at least document what the old values were. On the off chance we get someone who really cares, they likely only care on one backend (or we would have heard from them sooner!).